apps/judicial-system/backend/src/app/modules/case/case.controller.ts

```diff
@@ -790,12 +778,6 @@ export class CaseController {
   ): Promise<SigningServiceResponse> {
     this.logger.debug(`Requesting a signature for the ruling of case ${caseId}`)
 
-    if (user.id !== theCase.judgeId) {
```

Comment:
Moved to a new roles rule `districtCourtJudgeSignRulingRule` as permission checks should be in roles rules and guards when possible.